sources/osg/Matrix.js

```diff
@@ -652,9 +653,9 @@ var Matrix = {
     postMultRotate: ( function () {
         var r = matrixCreate();
         return function ( matrix, quat ) {
-            Matrix.makeRotateFromQuat( r, quat );
+            Matrix.makeRotateFromQuat( quat, r );
             Matrix.postMult( matrix, r );
-            return matrix;
+            return r;
```

Comment:
I'm gonna test now, but matrices don't have conmutative property, so if what you say is true, then is somethign too wrong in osgjs.